### DISCUSSION TITLE
Desgin Resources

### DISCUSSION THREAD

**Discussion by TinsFox** [tailwindlabs/tailwindcss#7913] [Help]
Do we have design resources? Something like [ant design](https://ant.design/docs/resources-cn). I really like the visual design of tailwindcss, but it is quite cumbersome to use tailwindcss to build the middle and backend. I want to use React to build a component library, but I don't know where to find the visual design specification.
I would be very grateful if you could tell me where to find the information I need

**Comment by adamwathan** [comment]
What do you mean by “visual design specification”? Tailwind is a framework for building custom designs, the design you end up with is totally up to you. There are no standard components or anything like Ant Design, the goal of Tailwind is to serve teams that need a way to implement their own designs rather than adopt an existing set of designed components.
